core/arch/arm/mm/tee_pager.c

```diff
@@ -622,12 +616,14 @@ static void tee_pager_save_page(struct tee_pager_pmem *pmem, uint32_t attr)
 
 	if (pmem->area->type == AREA_TYPE_RW && (attr & dirty_bits)) {
 		size_t offs = pmem->area->base & CORE_MMU_PGDIR_MASK;
-		size_t idx = pmem->pgidx - (offs >> SMALL_PAGE_SHIFT);
+		size_t fobj_pgidx = (pmem->pgidx - (offs >> SMALL_PAGE_SHIFT)) +
+				     pmem->area->fobj_pgidx;
```

Comment:
minor: fix indentation: 1 extra space here.

Reply:
Outdated change: feel free to ignore my comment.

Reply:
I'll fix